helm/hpcc/templates/esp.yaml

```diff
@@ -17,7 +17,7 @@ spec:
         accessDali: "yes"
         app: {{ $application }}
     spec:
-      {{- include "hpcc.placementMatchName" (dict "root" $ "pod" .name "cluster" .name "type" "esp") | indent 6 }}
+      {{- include "hpcc.placementsByPodType" (dict "root" $ "pod" .name "type" "esp") | indent 6 }}
```

Comment:
per comment in _heleprs.tpl, correct to have no "target" here, but better if unified function that can handle optional "target".